rskj-core/src/main/resources/reference.conf

```diff
@@ -121,7 +121,7 @@ peer {
     }
 
     # flag that allows to propagate a received block without executing it and only checking basic validation rules.
-    fastBlockPropagation = false
+    fastBlockPropagation = true
```

Comment:
it looks like there's no need to enable the feature in every config file. Doing this in `reference.conf` should be enough